go/engine/kex2_provisionee.go

```diff
@@ -595,6 +661,27 @@ func (e *Kex2Provisionee) cacheKeys() (err error) {
 	return e.ctx.LoginContext.SetCachedSecretKey(libkb.SecretKeyArg{KeyType: libkb.DeviceEncryptionKeyType}, e.dh, e.device)
 }
 
+func (e *Kex2Provisionee) storeEKs(ctx context.Context, deviceEKStatement keybase1.DeviceEkStatement, userEKBox *keybase1.UserEkBoxed) (err error) {
+	defer e.G().Trace("Kex2Provisionee.storeEKs", func() error { return err })()
```

Comment:
we should probably wrap this with a `ShouldRun` call until we're ready for key generation to go public

Reply:
done